test/testdata/lsp/definitions_multiple__a.rb

```diff
@@ -0,0 +1,17 @@
+# typed: true
+
+# Documentation 2
+class MyCommand
+  #   ^^^^^^^^^ def: MyCommand
+  #   ^^^^^^^^^ type-def: MyCommand
+  #   ^^^^^^^^^ hover: Documentation 1
+  #   ^^^^^^^^^ hover: Documentation 2
+end
+
+def main
+  MyCommand.new
+# ^^^^^^^^^ usage: MyCommand
+#           ^^^ type: MyCommand
+# ^^^^^^^^^ hover: Documentation 1
+# ^^^^^^^^^ hover: Documentation 2
```

Comment:
wow i hope no one is using this feature

Reply:
Primarily, this will fix the issue where the documentation is on an 'unlucky' loc.